vowpalwabbit/core/include/vw/core/reductions/cb/cb_explore_adf_common.h

```diff
@@ -105,6 +105,7 @@ class cb_explore_adf_base
       const multi_ex& ec_seq, VW::io::logger& logger);
 
   ExploreType explore;
+  static bool allow_multiple_costs;
```

Comment:
Does this need to be static?

Reply:
yes since it is used in predict/learn which are static

Reply:
If there are two workspace instances in the same process, one using cb with graph feedback and one not using it then this static would essentially break that scenario.

If this needs to be dynamic based on the setup then the variable and learn/predict should be non static

Reply:
actually it doesn't need to be static and good point, updated